qutebrowser/browser/adblock.py

```diff
@@ -173,15 +173,13 @@ def adblock_update(self):
         for url in config.val.content.host_blocking.lists:
             if url.scheme() == 'file':
                 filename = url.toLocalFile()
-                try:
-                    fileobj = open(filename, 'rb')
-                except OSError as e:
-                    message.error("adblock: Error while reading {}: {}".format(
-                        filename, e.strerror))
-                    continue
-                download = _FakeDownload(fileobj)
-                self._in_progress.append(download)
-                self._on_download_finished(download)
+                if os.path.isdir(filename):
+                    for filenames in os.scandir(filename):
+                        if not filenames.name.startswith('.')\
```

Comment:
I assume you have this check to exclude `.` and `..`? The `os.scandir` docs [say](https://docs.python.org/3.5/library/os.html#os.scandir):

> the special entries '.' and '..' are not included

So that'd only exclude names like `.hidden`, which I don't think we need to do.

Reply:
Also, please use parentheses around the expression instead of the `\` at the end of the line - inside parentheses, line breaks are ignored.

Reply:
I removed the unnecessary check, and i'll be sure to do that in the future.